contrib/PanopticSeg/paddlepanseg/core/train.py

```diff
@@ -92,6 +95,8 @@ def train(model,
         eval_ins (bool, optional): Whether or not to calculate instance segmentation metrics during validation. Default: False.
         precision (str, optional): If `precision` is 'fp16', enable automatic mixed precision training. Default: 'fp32'.
         amp_level (str, optional): The auto mixed precision level. Choices are 'O1' and 'O2'. Default: 'O1'.
+        profiler_options (str, optional): The option of train profiler.
+        to_static_training (bool, optional): Whether to use @to_static for training.
```

Comment:
`Whether to use @to_static for training.` -> `Whether or not to apply dynamic-to-static model training.`